hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/amrmproxy/AMRMProxyApplicationContextImpl.java

```diff
@@ -53,7 +53,7 @@ public class AMRMProxyApplicationContextImpl implements
    * @param nmContext NM context
    * @param conf configuration
    * @param applicationAttemptId attempt id
-   * @param user user name of the application
+   * @param user username of the application
```

Comment:
I would leave this as is.

Reply:
Thank you very much for reviewing the code! I will fix it.